scripts/exregional_run_analysis.sh

```diff
@@ -424,18 +424,27 @@ if [ ${BKTYPE} -eq 1 ]; then  # cold start uses background from INPUT
 else                          # cycle uses background from restart
   if [ "${IO_LAYOUT_Y}" == "1" ]; then
     ln_vrfy  -snf ${bkpath}/fv_core.res.tile1.nc             fv3_dynvars
-    ln_vrfy  -snf ${bkpath}/fv_tracer.res.tile1.nc           fv3_tracer
+    if [ ${anav_type} == "AERO" ]; then
+      cp_vrfy ${bkpath}/fv_tracer.res.tile1.nc               fv3_tracer
```

Comment:
why aerosol DA needs to copy the tracer file instead of a link?

Reply:
@guoqing-noaa 
This change was in the previous PR. 
GSI PM2.5 DA is a separate task, and can be running parallel with the meteorological DA. This is to avoid the two GSI runs write into a same trace file at the same time.

Reply:
Did we already merged this PR? A new PR need start from the current RRFS_dev1 head.

Reply:
yes, there were merged in.  
PR today is to fix dependence issue on GSI SD in postanal tasks. Please check details in the commit today. Thanks.

Reply:
@hongli-wang Thanks for the reply.

I agree with Ming that this PR has lots of unnecessary commit history.
Could you base your PR off the latest feature/RRFS_dev1 branch?

Thanks!